resources/assets/js/mix/trainingticket.js

```diff
@@ -44,41 +44,48 @@ if (newForm.length || (editForm.length && draft.length)) {
   }, 60000);
 }
 
-var stars = $("#stars span");
+const stars = document.querySelectorAll("#stars span");
 
 // Needed for if you submit and get redirected back the stars will now still be highlighted
-var rating = $("#stars input[name='score']").val();
+var rating = document.querySelector("#stars input[name='score']").value;
 if (rating) {
   for (var i = 0; i < rating; i++) {
     stars.eq(i).text("\u2605");
   }
 }
 
-stars.each(function () {
-  $(this).hover(
-    function () {
-      $(this).prevAll().addBack().addClass("star-hover");
-    },
-    function () {
-      $(this).prevAll().addBack().removeClass("star-hover");
+document.querySelectorAll(".your-selector").forEach(function (el) {
+  el.addEventListener("click", function () {
+    // Reset all stars to hollow (☆)
+    document.querySelectorAll("#stars .your-selector").forEach(function (star) {
```

Comment:
Again here we already have `stars` defined, so we might as well use it

Reply:
If you're getting an error that `stars` doesn't exist, you may have to pass it into the scope somehow. I think I've had this issue with JS before